src/SwaggerProvider.DesignTime/Utils.fs

```diff
@@ -29,16 +38,18 @@ module SchemaReader =
               } |> Async.Catch
             match res with
             | Choice1Of2 x -> return x
-            | Choice2Of2 (:? System.Net.WebException as wex) when wex.Response <> null ->
+            | Choice2Of2 (:? System.Net.WebException as wex) when not <| isNull wex.Response ->
                 use stream = wex.Response.GetResponseStream()
-                use reader = new System.IO.StreamReader(stream)
+                use reader = new StreamReader(stream)
                 let err = reader.ReadToEnd()
                 return 
                   if String.IsNullOrEmpty err then raise wex
                   else err.ToString()
             | Choice2Of2 e -> return failwith(e.ToString())
-        | false ->
-            use sr = new System.IO.StreamReader(schemaPathRaw)
+        | _ ->
+            let request = WebRequest.Create(schemaPathRaw)
```

Comment:
@xperiandri why do you use `WebRequest` to load file content from hard drive?

Reply:
Because it is Universal and can handle FTP and whatever other protocol you want and register.
File and FTP includes out of the box